### ISSUE TITLE
Why the train_batch.jpg have no labels plotting

### ISSUE THREAD

**Issue by womengjianhai** [ultralytics/yolov5#5497]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Training

### Bug

For my own datasets, which is small and the bounding box is slim. I find the process files during training have no labels plotting , expecially i set mosaic to 0.0.

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by womengjianhai** [comment]
Also, if i keep mosaic augmentation, i find some labels of part of images are hidden.

**Comment by womengjianhai** [comment]
the val process， "labels" in the log is zero most of the time.

**Comment by glenn-jocher** [comment]
@womengjianhai see **Train Custom Data** tutorial to get started and for indications on how to verify your labels are correct. 

![image](https://user-images.githubusercontent.com/26833433/140313046-8c954fa7-52ee-4863-a742-0126906e19a4.png)

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
